onnxruntime/core/providers/cpu/reduction/reduction_ops.cc

```diff
@@ -53,11 +53,13 @@ bool PrepareForReduce(OpKernelContext* ctx,
   const Tensor& input = *input_tensor_ptr;
 
   size_t ndim = input.Shape().GetDims().size();
-  for (int64_t axe : axes_) {
-    ORT_ENFORCE(axe >= 0 && axe < (int64_t)ndim, "Axis attribute out of range");
+  std::vector<int64_t> axes = axes_;
+  for (size_t i = 0; i < axes.size(); i++) {
+    if (axes[i] < 0)
+      axes[i] += ndim;
+    ORT_ENFORCE(axes[i] >= 0 && axes[i] < (int64_t)ndim, "Axis attribute out of range");
   }
```

Comment:
common.h in the providers project has a helper method HandleNegativeAxis to do this. can you use that?

Reply:
done, thx!